tests/PySys/call_tedge_connect_c8y_--test_positive/run.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+"""
+Test `tedge connect c8y --test` (successful case):
+
+Given a configured system with configured certificate
+When we setup EnvironmentC8y
+When we execute `sudo tedge connect c8y --test`
+When we validate stdout
+When we cleanup EnvironmentC8y
+Then we find a successful message in stdout
+Then the test has passed
+
+"""
+
+
+class RestartBridge(EnvironmentC8y):
+    def setup(self):
```

Comment:
Just FYI: If you don't really have any explicit "setup" logic, redeclaring the setup method here is optional. I see that you're registering a cleanup function in this setup. But, that cleanup is not doing anything either. So you could skip both these `setup` and `mycleanup` functions.

Reply:
good to know. Removed these two classes.